pkg/apis/application/v1alpha1/app_types.go

```diff
@@ -97,6 +101,14 @@ type AppSpecConfig struct {
 	Secret AppSpecConfigSecret `json:"secret,omitempty"`
 }
 
+// +k8s:openapi-gen=true
+type AppSpecInstall struct {
+	// +kubebuilder:validation:Optional
+	// +nullable
+	// SkipCRDs when true decides that CRDs which are supplied with the chart are not installed. Default: false.
+	SkipCRDs bool `json:"skipCrds,omitempty"`
```

Comment:
```suggestion
	SkipCRDs bool `json:"skipCRDs,omitempty"`
```

@yasn77 Could we make the JSON tag upper case as well? We don't have many abbreviations in CRDs but I think they're usually upper case.